.cirrus.yml

```diff
@@ -93,7 +94,7 @@ smoke_task:
     name: "Smoke Test"
 
     gce_instance:
-        memory: "12Gb"
```

Comment:
pre existing but defining a task with no cpu seems odd

Reply:
Sure.  We'll try 4, maybe it'll run this task a bit faster.